lib/css-extract-extends.js

```diff
@@ -4,7 +4,7 @@ var makeComposition = require('./composition').makeComposition;
 
 var regex = /\.([^\s]+)(\s+)(extends\s+)(\.[^{]+)/g;
 
-module.exports = function extractExtends(css, hashed) {
```

Comment:
Yeah, this second argument isn't even used anymore. Somehow I forgot to take this out.